src/corner_kick/src/modules/Logger/index.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Subscribe } from 'unstated';
+
+import {
+  TopicListener,
+  TopicListenerType,
+} from '~/components/containers/TopicListener';
+import { IROSOut } from '~/types/standardTopics';
+import { LogMessage } from './LogMessage';
+
+/**
+ * Logging module. Displays the /rosout ROS topic, which is used by all nodes
+ * to display debugging information.
+ */
+export const Logger = () => {
+  // Here, we subscribe to the /rosout topic. The function mapLogMessages will
+  // automatically be called when
+  // a new message is sent to /rosout, allowing us to update our UI.
+  return (
+    <Subscribe to={[TopicListener('/rosout')]}>{mapLogMessages}</Subscribe>
```

Comment:
Topic here should maybe be in the constants file? Not too sure about this one. I'm ok with it either way, your call.